package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@hmcts/feature-toggle-client",
+  "version": "1.3.0",
+  "description": "JavaScript client for feature togggle",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "scripts": {
+    "prepublishOnly": "tsc -p tsconfig.publish.json",
+    "postpublish": "rm -rf dist/",
+    "lint": "tslint --project tsconfig.json",
+    "test": "NODE_ENV=mocha LOG_LEVEL=OFF mocha --opts mocha.opts --reporter-options reportFilename=unit,inlineAssets=true,reportTitle=feature-toggle-client 'src/test/client/**/*.ts'",
+    "test:coverage": "NODE_ENV=mocha LOG_LEVEL=OFF nyc mocha --opts mocha.opts 'src/test/client/**/*.ts'",
+    "test:codacy-upload": "cat ./coverage-report/lcov.info | codacy-coverage",
+    "test:codecov-upload": "nyc report --reporter=json && codecov -f coverage-report/*.json",
+    "precommit": "yarn lint",
+    "prepush": "yarn test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@git.github.com:hmcts/feature-toggle-client.git"
+  },
+  "author": "HMCTS Reform",
+  "license": "MIT",
+  "dependencies": {
+    "codacy-coverage": "^2.0.3",
+    "codecov": "^3.0.0",
+    "request": "^2.81.0",
+    "request-promise-native": "^1.0.3"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.3",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/chai-spies": "^0.0.0",
+    "@types/http-status-codes": "^1.0.29",
+    "@types/lodash": "^4.14.78",
+    "@types/mocha": "*",
+    "@types/nock": "^8.2.1",
+    "@types/node": "^8.0.24",
+    "@types/request-promise-native": "^1.0.2",
+    "@types/sinon-chai": "^2.7.28",
+    "@types/supertest": "*",
+    "chai": "^4.1.1",
+    "chai-as-promised": "^7.1.1",
+    "cookie": "^0.3.1",
+    "debug": "^3.0.0",
+    "http-status-codes": "^1.3.0",
+    "husky": "^0.14.3",
+    "lodash": "^4.17.4",
+    "mocha": "^4.0.1",
+    "mochawesome": "^2.2.0",
+    "mock-require": "^2.0.1",
```

Comment:
I do not think we need that. If so please remove.

Reply:
Removed - Also removed a bunch of other modules that weren't needed.